stunnel/os_family_map.yaml

```diff
@@ -13,6 +13,7 @@ Debian:
   pid_dir: /var/run/stunnel4
   use_chroot: true
   certificates_auto_config: true
+  clean_conf_dir: false
```

Comment:
this file is for OS specific settings, better set the default value in `defaults.yaml`